source/braille.py

```diff
@@ -387,50 +393,50 @@
 BLUETOOTH_PORT = ("bluetooth", _("Bluetooth"))
 
 
-class formattingSymbol(NamedTuple):
-	attribute: str
-	startSymbol: str
-	endSymbol: str
+class FormattingMarker(NamedTuple):
+	"""A pair of braille symbols that indicate the start and end of a particular type of font formatting.
+
+	As these are shapes, they should be provided in unicode braille.
+	"""
+
+	start: str
+	end: str
 
 
-fontAttributeBrailleTags: tuple[formattingSymbol] = (
-	formattingSymbol(
-		attribute="bold",
+fontAttributeFormattingMarkers: dict[str, FormattingMarker] = {
+	"bold": FormattingMarker(
```

Comment:
does "bold" need to be a string? Otherwise I would go with making fontAttributeFormattingMarkers an Enum 

e.g. the earlier suggestion

```py
class FormattingSymbol(FormattingMarker, Enum):
	BOLD = FormattingMarker(
		# Translators: Brailled at the start of bold text.
		# This is the English letter "B" in braille.
		start=pgettext("braille formatting symbol", "⠃"),
		# Translators: Brailled at the end of bold text.
		# This is the English letter "B" plus dot 7 in braille.
		end=pgettext("braille formatting symbol", "⡃"),
	)
```

Reply:
We're using the key as a lookup into the `field` and `fieldCache`, which is a `dict[str, str]`. Making this an enum would require more complex logic to look up values in the format fields, so I believe it should stay as a `dict[str, FormattingMarker]`.

Reply:
makes sense, was just confirming